wasm/jit/jit_arm64.s

```diff
@@ -0,0 +1,11 @@
+//go:build arm64
+// +build arm64
+
+#include "funcdata.h"
+#include "textflag.h"
+
+// jitcall(codeSegment, engine)
+TEXT ·jitcall(SB),NOSPLIT|NOFRAME,$0-16
+        MOVD codeSegment+0(FP),R1  // Load the address of native code.
+        MOVD engine+8(FP),R0       // Load the address of engine.
```

Comment:
only difference from amd64.s is that MOVD is for 64bit regs with arm64 assembler, not MOVQ as in amd64